doc/input-output-reference/src/overview/group-demand-limiting-controls.tex

```diff
@@ -70,15 +70,15 @@ \subsubsection{Inputs}
 
 This field indicates what logic should be used to activate the individual DemandManagers in the list. \textbf{Sequential} treats the list of DemandManagers as a sequential priority list, i.e., the first DemandManager in the list is activated first, then the second, etc., until the demand falls below the limit. \textbf{All} activates all demand managers to achieve the maximum demand reduction.
 
-\paragraph{Field: Demand Manager Type 1}\label{field-demand-manager-type-1}
+\paragraph{Field: Demand Manager \textless{}\#\textgreater{} Type}\label{field-demand-manager-type-1}
 
 The key word defining the type of DemandManager.
 
-\paragraph{Field: Demand Manager Name 1}\label{field-demand-manager-name-1}
+\paragraph{Field: Demand Manager \textless{}\#\textgreater{} Name}\label{field-demand-manager-name-1}
 
 The name of a DemandManager object defined elsewhere in the input file.
 
-DemandManagers are listed by pairs of data items: \emph{Demand Manager Type} and \emph{Demand Manager Name}. Ten managers are accommodated  in the list by default. The IDD specification, however, is extensible and additional pairs may be added by directly editing the IDD.
+DemandManagers are listed by pairs of data items: \emph{Demand Manager Type} and \emph{Demand Manager Name}. These two fields are extensible, so up to 10 additional pairs of these fields can be added to the end of this object.
```

Comment:
This object is extensible, so ~up to 10~ additional pairs of these fields ....

Reply:
Got it--fixed in the next commit.